tests/unit/test_column_noise.py

```diff
@@ -749,22 +860,48 @@ def test_generate_phonetic_errors(dummy_dataset, column):
         NOISE_TYPES.make_phonetic_errors.name
     ]
     noised_data = NOISE_TYPES.make_phonetic_errors(
-        dummy_dataset, config, RANDOMNESS0, "dataset", column
+        data, config, RANDOMNESS0, "dataset", column
     )
     data = data.squeeze()
 
     # Validate we do not change any missing data
-    missing_mask = data.isna()
-    assert noised_data[missing_mask].isna().all()
+    missing_mask = data == ""
+    assert (noised_data[missing_mask] == "").all()
 
     # Check expected noise level
-    cell_prob = config[Keys.CELL_PROBABILITY]
+    cell_probability = config[Keys.CELL_PROBABILITY]
+    token_probability = config[Keys.TOKEN_PROBABILITY]
     check_original = data[~missing_mask]
     check_noised = noised_data[~missing_mask]
-    actual_noise = (check_original != check_noised).mean()
-    # We are setting lower and upper bounds for testing
-    assert actual_noise < cell_prob
-    assert actual_noise > cell_prob / 10
+    actual_noise = (check_original != check_noised).sum()
+    first_names = [
```

Comment:
Again, might want to store these as constants.

I plan to make these longer/more comprehensive in the bugfix PR.

Reply:
See above comment. When you add longer/more comprehensive tests for OCR and phonetic, you may need to mark them as slow depending on their performance.